node/clients/profile/profileV2.ts

```diff
@@ -219,7 +219,7 @@ export class ProfileClientV2 extends JanusClient {
           postalCode: address.postalCode,
           profileId: address.userId,
           route: address.street,
-          streetNumber: address.number,
+          streetNumber: address.number || '',
```

Comment:
Prefer using nullish coalescing operator (`??`) instead of a logical or (`||`), as it is a safer operator.eslint[@typescript-eslint/prefer-nullish-coalescing](https://github.com/typescript-eslint/typescript-eslint/blob/v3.10.1/packages/eslint-plugin/docs/rules/prefer-nullish-coalescing.md)

```suggestion
          streetNumber: address.number ?? '',
```

Reply:
Thanks for the tip, @emersonlaurentino!